src/rpdk/core/data_loaders.py

```diff
@@ -156,6 +156,12 @@ def load_resource_spec(resource_spec_file):  # pylint: disable=R # noqa: C901
         "maxItems",
         "exclusiveMinimum",
         "exclusiveMaximum",
+        "additionalItems",
+        "additionalProperties",
+        "uniqueItems",
+        "pattern",
+        "patternProperties",
+        "multipleOf",
     }
     try:  # pylint: disable=R
         for _key, schema in JsonSchemaFlattener(resource_spec).flatten_schema().items():
```

Comment:
if u are not using `_key` could u change it to `_`

Reply:
just to clarify - you are using flattened schema to avoid recursive traversal, since all the subschemas are on the same level?